Dockerfile

```diff
@@ -17,12 +17,8 @@ RUN apt-get update -qq \
 # Copy entire bundle to the working directory
 COPY . .
 
-# Define base enviroment
-ENV UPLOAD_USER user
-ENV UPLOAD_PASSWORD hackme
```

Comment:
When the image is run without these env variables, the output is quite ugly (seems to come from the auth library?). Is there anyway you can do a check that these variables are set when the server starts and print a nice error if they're not?